src/langchain_google_alloydb_pg/vectorstore.py

```diff
@@ -349,6 +351,39 @@ async def aadd_documents(
         ids = await self.aadd_texts(texts, metadatas=metadatas, ids=ids, **kwargs)
         return ids
 
+    def encode_image(self, uri: str) -> str:
+        """Get base64 string from image URI."""
+        with open(uri, "rb") as image_file:
+            return base64.b64encode(image_file.read()).decode("utf-8")
+
+    async def aadd_images(
+        self,
+        images: List[str],
+        metadatas: Optional[List[dict]] = None,
+        ids: Optional[List[str]] = None,
+        **kwargs: Any,
+    ) -> List[str]:
+        """Embed images and add to the table."""
+        encoded_images = []
+        if metadatas is None:
+            metadatas = [{"image_uri": uri} for uri in images]
+
+        for uri in images:
+            encoded_image = self.encode_image(uri)
```

Comment:
Right now we don't support GCS uris since we are dowloading everything, correct?

Reply:
Right now we support local URI for all embedding services, and GCS URI for Vertex AI embedding because its `embed_image()` implementation accepts GCS URI input: https://github.com/langchain-ai/langchain-google/blob/72d1b5ad853dd1d80c5f74089290caba712221fe/libs/vertexai/langchain_google_vertexai/embeddings.py#L470